docs/netobserv_cli.adoc

```diff
@@ -1,9 +1,12 @@
 // Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
 :_mod-docs-content-type: REFERENCE
-[id="network-observability-cli-usage_{context}"]
+
+[id="network-observability-netobserv-cli-reference_{context}"]
 = Network Observability CLI usage
 
-You can use the The Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flow data and packet data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and flowlogs-pipeline.
+You can use the The Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flows data and packets data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and `flowlogs-pipeline`.
```

Comment:
```suggestion
You can use the Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flows data and packets data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and `flowlogs-pipeline`.
```

Reply:
seems like this been introduced in the other pr here:
https://github.com/netobserv/network-observability-cli/pull/146/commits/145cc5dc923b62289c91f1a3c56b767af4974962